rpm.go

```diff
@@ -426,34 +432,69 @@ func (r *RPM) writeGenIndexes(h *index) {
 	// rpm utilities look for the sourcerpm tag to deduce if this is not a source rpm (if it has a sourcerpm,
 	// it is NOT a source rpm).
 	h.Add(tagSourceRPM, EntryString(fmt.Sprintf("%s-%s.src.rpm", r.Name, r.FullVersion())))
-	if r.pretrans != "" {
-		h.Add(tagPretrans, EntryString(r.pretrans))
-		h.Add(tagPretransProg, EntryString("/bin/sh"))
+}
+
+func (r *RPM) writeSingleScriptlet(h *index, scriptlet, name string, scriptletTag, scriptletProgramTag int) error {
+	if scriptlet != "" {
```

Comment:
Prefer using an early return to avoid this indentation. i.e.

```
if scriptlet == "" {
  return nil
}
```

Reply:
Done